ex1/ex1Spec.js

```diff
@@ -0,0 +1,19 @@
+var getThirdCommaIndex = require("./ex1");
+
+describe("getThirdCommaIndex", function() {
+
+	it("returns -1 when no third comma", function() {
+		expect(getThirdCommaIndex('hel,lowo,rld', ','))
+		.toBe(-1);
+	});
+
+	it("finds a comma in a regular sentence", function() {
+		expect(getThirdCommaIndex('hello,world,this,is,a,great,day', ','))
+		.toBe(16);
+	});
+
+	it("works even when there are no characters beside commas", function() {
+		expect(getThirdCommaIndex(',,,', ','))
+		.toBe(2);
```

Comment:
When you do break a line because it is long, you need to push the continuation line in: +tab.